application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -90,6 +97,14 @@ private void handleHelpThreadCreated(ThreadChannel threadChannel) {
             .queueAfter(5, TimeUnit.SECONDS);
     }
 
+    private static User getMentionedAuthorByMessage(Message message) {
+        return message.getMentions().getUsers().get(0);
+    }
+
+    private boolean isPostedByBot(ThreadChannel channel, Message message) {
```

Comment:
confusing name. name implies it would check for any bot, but it actually only checks for self user. rename to `isPostedBySelfUser`